manager/vulnerabilities_handler.py

```diff
@@ -26,14 +26,15 @@ def __init__(self, list_args, query_args, args={}): # pylint: disable=dangerous-
                     CveMetadata.cve.alias("cve_name"),
                     fn.Min(VulnerabilitySource.name).alias("source_name"),
                     CveMetadata.cvss3_score,
-                    CveMetadata.impact_id,
+                    CveMetadata.cvss2_score,
+                    CveImpact.name.alias("impact"),
```

Comment:
This lane breaks #94,  this should be reverted back to CveMetadata.impact_id

Reply:
ok, I will fix in the same commit with pylint fixes. then request a review from you.